code/drasil-example/Drasil/Projectile/Assumptions.hs

```diff
@@ -78,10 +77,6 @@ pointMassDesc :: Sentence
 pointMassDesc = (S "size" `sAnd` S "shape") `ofThe'` phrase projectile `sAre`
                 S "negligible" `sC` S "so that it can be modelled as a point" +:+. phrase mass
 
-neglectCurvDesc :: Sentence
-neglectCurvDesc = S "The" +:+ phrase distance `sIs` S "small enough that" +:+.
-                  (S "curvature" `ofThe` S "Earth can be neglected")
-
```

Comment:
No, we shouldn't remove this assumption.  For one reason because @JacquesCarette likes this assumption.  :smile:  For another, better reason, removing this assumption is removing information.  It is true that we have the assumption that the height of the launcher and the target are the same.  We also have the assumption that we can use a Cartesian coordinate system.  These assumptions are possible, in part, because we have assumed that we can ignore the curvature of the earth.  These are the dependencies between assumptions that have been mentioned elsewhere.  Instead of removing assumptions, we should keep them all and explain the relationships between them.  In a different problem context some assumptions will drop out and others will stay.  Since we don't know what family members we might want in the future, we should keep all assumptions.

Reply:
Added back in 2807130